libsrc/apple2/crt0.s

```diff
@@ -58,7 +58,7 @@ init:   ldx     #zpspace-1
         ; Check for ProDOS.
         ldy     $BF00           ; MLI call entry point
         cpy     #$4C            ; Is MLI present? (JMP opcode)
```

Comment:
I'm not perfectly sure but could it be possible to save the result of this check via PHP and restore it later via PLP so that we can get it cheaply further down where we have the LC write-enabled anyway?